### ISSUE TITLE
[Bug]: Issue with Portsmouth UK returning an error

### ISSUE THREAD

**Issue by snigehere** [mampfes/hacs_waste_collection_schedule#1440]
### I Have A Problem With:

A specific source

### What's Your Problem

I have been using the integration for a while - a couple of days ago I saw that the sensor was reporting as unavailable.  Checking I see errors reported in the logs.  UPRN is 1775062139

I am not a code person so have not been able to construct the URL to be able to test this but have been onto the portal and am able to find my collection schedule following the site prompts.

Hope this is helpful

### Source (if relevant)

portsmouth_gov_uk.py

### Logs

```Shell
2023-11-21 21:56:43.826 ERROR (SyncWorker_2) [waste_collection_schedule.source_shell] fetch failed for source Portsmouth City Council:
Traceback (most recent call last):
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch
    entries = self._source.fetch()
              ^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/portsmouth_gov_uk.py", line 64, in fetch
    date = datetime.strptime(date, '%A %d %B %Y').date(),
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/_strptime.py", line 352, in _strptime
    raise ValueError("unconverted data remains: %s" %
ValueError: unconverted data remains: *
```

### Relevant Configuration

```YAML
# waste collection schedule
waste_collection_schedule:
  sources:
    - name: portsmouth_gov_uk
      args:
        uprn: 1775062139
      customize:
        - type: recycling bin
          icon: mdi:recycle
        - type: refuse bin
          icon: mdi:trash-can
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
fixed by #1442 
will be part of release 1.44.0 or is available now as master (git) version (HACS: 3-dot menu -> re-download -> change version to master)

**Comment by snigehere** [comment]
Many thanks @5ila5 for your fast action which is very much appreciated. Master downloaded and all working.
